src/hotspot/share/memory/oopFactory.cpp

```diff
@@ -39,7 +39,7 @@
 #include "oops/oop.inline.hpp"
 #include "oops/typeArrayKlass.hpp"
 #include "oops/typeArrayOop.inline.hpp"
-#include "oops/valueKlass.hpp"
+#include "oops/inlineKlass.hpp"
```

Comment:
Do we really need this include?
InlineKlass is not used directly in this file, ValueArrayKlass is used and valueArrayKlass.hpp includes inlineKlass.hpp.